packages/react-dropdown/package.json

```diff
@@ -30,6 +30,7 @@
     "@emotion/core": "^10.0.6",
     "@emotion/css": "^10.0.6",
     "@emotion/styled": "^10.0.6",
+    "@emotion/styled-base": "10.0.4",
```

Comment:
Why do we need this new dependency?

Reply:
this is so that the ui.quid.com interface can properly toggle between light and dark theme for the dropdown component. previously the dropdown component was showing as light mode regardless of what the user toggled. cc: @gabrielmicko